### PR TITLE
Change license to BSD 3-clause

### PR DESCRIPTION
**Pull request by czthiele**
- move from BSD 4-clause to BSD 3-clause since all other projects are using this license
- BSD 3-clause does not contain the controversial "advertising clause"
- removing that clause seems ok since the API is only useful in combination with a PiWeb license

### CODE REVIEW

**Comment by czthiele** [comment]
Hi @czwinter, together with @georg-eckert-zeiss we decided that BSD 3-clause seems to be the better choice, since all other repositories also use this license. This removes the controversial "advertising clause". I think this proposes no disadvantage, since any reasonable usage of our C# NuGet always implies a valid PiWeb license on the users side. Is this ok from your point of view?

File: LICENSE.md [hunk 1]
```diff
@@ -1,16 +1,31 @@
+BSD 3-Clause License
+
 LICENSE
 =======
-Copyright (c) 2014, Carl Zeiss Industrielle Messtechnik GmbH
+Copyright (c) 2014, Carl ZEISS Industrielle Messtechnik GmbH
```
Comment (czwinter) [inline LICENSE.md:5]: In the context of the legal entity Zeiss should not be used upper case: "Carl Zeiss Industrielle Messtechnik GmbH".

File: src/License.txt [hunk 2]
```diff
@@ -1,24 +1,29 @@
-﻿Copyright (c) 2014, Carl Zeiss Industrielle Messtechnik GmbH
+﻿BSD 3-Clause License
 
+Copyright (c) 2014, Carl ZEISS Industrielle Messtechnik GmbH
```
Comment (czwinter) [inline src/License.txt:3]: Zeiss

**Review by czwinter — APPROVED** [review]

File: LICENSE.md (hunk 1 above)
Comment (georg-eckert-zeiss) [reply LICENSE.md]: There is a document on that. I'll send you a copy.

File: LICENSE.md [hunk 3]
```diff
@@ -1,16 +1,31 @@
+BSD 3-Clause License
```
Comment (georg-eckert-zeiss) [inline LICENSE.md:1]: Use either short identifier (BSD-3-Clause) or Full name (BSD 3-Clause "New" or "Revised" License)

**Review by georg-eckert-zeiss — CHANGES_REQUESTED** [review]

File: src/License.txt [hunk 4]
```diff
@@ -1,24 +1,29 @@
-﻿Copyright (c) 2014, Carl Zeiss Industrielle Messtechnik GmbH
+﻿BSD 3-Clause License
```
[the remark this answers is not in the document]
Comment (georg-eckert-zeiss) [reply src/License.txt:1]: We discussed to use only one License file in the repo root and draw it into the NuGet from there.

File: LICENSE.md (hunk 3 above)
Comment (georg-eckert-zeiss) [reply LICENSE.md:1]: I recommend the short identifier.

Comment (czthiele) [reply LICENSE.md]: Done

File: src/License.txt (hunk 4 above)
[the remark this answers is not in the document]
Comment (czthiele) [reply src/License.txt]: Done

File: LICENSE.md (hunk 3 above)
Comment (georg-eckert-zeiss) [reply LICENSE.md]: This

**Review by georg-eckert-zeiss — APPROVED** [review]

**Review by georg-eckert-zeiss — APPROVED** [review]

**Pull request merged**